packages/api/cms-api/src/redirects/redirects.resolver.ts

```diff
@@ -70,6 +72,8 @@ export function createRedirectsResolver({
                 options.orderBy = { [sortColumnName]: sortDirection };
             }
 
+            await this.pageTreeReadApi.preloadNodes(scope);
```

Comment:
Does this work as intended? The redirect scope is `{ domain: "main" }`, but the page tree node scope is `{ domain: "main", language: "en" }`. Will this load all nodes for the domain and all languages, or will this load _all_ nodes?

Reply:
yes, this works as intended. It preloads PageTree with all languages, but just for one domain.